### PR TITLE
Fix compliance policy deployment link

### PR DESCRIPTION
**Pull request by adamruzicka**
CC @pondrejk

### CODE REVIEW

**Comment by adamruzicka** [comment]
Also cc @lennonka

**Review by Lennonka — APPROVED** [review]
The link does have a target. However, I'm not sure about its context of origin and, therefore, I cannot evaluate if this target is appropriate.

According to the key of the link, you might want to consider `/managing_security_compliance/deploying-compliance-policies_security-compliance`, but I'm not blocking it.

**Comment by adamruzicka** [comment]
It's used in Hosts > Compliance > Policies > New Policy 
image

We already had this in one of the downstream PRs before the theme plugin got upstreamed. If you say the other link is better, then I'll gladly switch to it.

**Comment by Lennonka** [comment]
Then it's already the best possible option!

**Review by pondrejk — APPROVED** [review]

File: lib/foreman_theme_satellite/documentation.rb [hunk 1]
```diff
@@ -69,8 +69,8 @@ module Documentation
       'Managing_Compliance_Policies_security-compliance' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/Managing_Compliance_Policies_security-compliance",
       'Configuring_SCAP_Contents_security-compliance' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/Configuring_SCAP_Contents_security-compliance",
       'tailoring-xccdf-profiles_security-compliance' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/Configuring_SCAP_Contents_security-compliance#tailoring-xccdf-profiles_security-compliance",
-      'deploying-compliance-policies_security-compliance' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/Configuring_SCAP_Contents_security-compliance#deploying-compliance-policies_security-compliance",
-    }.freeze
+      'deploying-compliance-policies_security-compliance' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/compliance-policy-deployment-options_security-compliance",
+    }
```
Comment (evgeni) [inline lib/foreman_theme_satellite/documentation.rb]: Any reason you dropped the `.freeze`?

Comment (adamruzicka) [reply lib/foreman_theme_satellite/documentation.rb]: Fat fingers I'm afraid, frozen again.

**Review by evgeni — APPROVED** [review]

**Pull request merged**